### ISSUE TITLE
perceptual loss pretrained

### ISSUE THREAD

**Issue by fishman2008** [KupynOrest/DeblurGAN#77]
It appears that perceptual loss does not set the requires_grad= False. Any impact on that? Does that mean the gradients are still got updated? Another question is that if I need to access the features located in classifier (e.g., last layer before final FC), other than the conv_3_3 in features module, what will be the code to finish that? Thanks.

**Comment by KupynOrest** [comment]
Hello, thanks for the question, I'll have to check it, as described in Style Transfer works, the network is set to `eval()` mode before computing the loss, although I believe that shouldn't change anything in terms of reaching the final optimum.
To change the layer, you can simply specify the need one in losses.py script ither by number or the name, you can see all the available layers here - https://github.com/pytorch/vision/blob/master/torchvision/models/vgg.py
